test/suites/mask_fromjson.yml

```
@@ -0,0 +1,38 @@
+name: fromjson features
+testcases:
+  - name: null value
+    steps:
+      - script: cp ../masking.yml masking.yml
+      - script: |-
+          echo '{"sourcefield": "null", "targetfield": ""}' | pimo
+        assertions:
+          - result.code ShouldEqual 0
+          - result.systemout ShouldEqual {"sourcefield": "null", "targetfield": null}
```

Ici il faudra bien faire attention à ne pas utiliser d'espaces dans l'attendu, PIMO sort du JSON qui ne contient aucun espace.

```suggestion
          - result.systemout ShouldEqual {"sourcefield":"null","targetfield":null}
```

---

C'est à faire sur tous les tests du fichier bien sûr :)